languages.toml

```diff
@@ -277,6 +277,15 @@ roots = []
 comment-token = "--"
 indent = { tab-width = 2, unit = "  " }
 
+[[language]]
+name = "svelte"
+scope = "source.svelte"
+injection-regex = "svelte"
+file-types = ["svelte"]
+roots = []
+indent = { tab-width = 2, unit = "  " }
+language-server = { command = "svelteserver" }
```

Comment:
Did you test this? It probably needs `args = ["--stdio"]` based on what I've seen elsewhere

Reply:
Not yet, I wanted to make a prospective PR to see if I had misunderstood things.
it seems you are right here
https://github.com/sveltejs/language-tools/blob/master/packages/language-server/src/server.ts#L67

thank you for the review!